firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nBaseSnapshotTests.swift

```diff
@@ -140,4 +140,11 @@ extension XCUIElement {
         L10nBaseSnapshotTests().mozWaitForElementToExist(self, timeout: timeout)
         self.tap()
     }
+
+    /// Waits for the UI element and then taps and types the provided text if it exists.
+    func tapAndTypeText(_ text: String, timeout: TimeInterval? = TIMEOUT) {
```

Comment:
Fix an existing compilation issue. A recent addition `tapAndTypeText` has been introduced in `BaseTestCase` but it is not propagated to `L10nBaseSnapshotTests` class.